src/index.tsx

```diff
@@ -48,6 +53,94 @@ function Updaters() {
     )
 }
 
+const Input = styled.input<{ error?: boolean }>`
+    border-radius: 5px;
+    width: 100%;
+`
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 30px;
+    padding: 10%;
+    justify-content: center;
+    align-items: center;
+`
+
+function CustomApp() {
+    const [auth, setAuth] = useState(false)
+    const PASSWORD = 'gdbetatest'
+    const PASSWORD_HASH = '$2a$10$V9DPoPvZtRpg9t23wzl5c.jYYyG5VJdJx/pvBJy61WmN/01rkSfSm'
+
+    useEffect(() => {
+        const passMatch = window.location.search.match(/\bpass=\w+,\b/)
+        if (passMatch && passMatch[0]) {
+            let pass = passMatch[0].split('=')[1]
+            pass = pass.substring(0, pass.length - 1)
+
+            if (pass === PASSWORD) setAuth(true)
```

Comment:
set localstorage for password found in url, so if user comes back but without password he is still logged in

Reply:
you didnt change it so that it is saved in localstorage.
`so if user comes back but without password he is still logged in`